config/default.yml

```diff
@@ -219,7 +219,7 @@ RSpec/ExampleWording:
 RSpec/ExpectChange:
   EnforcedStyle: 'block'
 
-RSpec/FactoryBot/SyntaxMethods:
+FactoryBot/SyntaxMethods:
```

Comment:
This namespace change [is new as of 2.22.0](https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md#2220-2023-05-06) -- to avoid breaking clients who are on older `rubocop-rspec` versions, we should update the gemspec to `>= 2.22` for the `rubocop-rspec` dependency.